bciers/apps/reporting/src/app/demo/page.tsx

```diff
@@ -1,3 +1,57 @@
+import { TaskListElement } from "@bciers/components/navigation/reportingTaskList/types";
+import ReportingTaskList from "@bciers/components/navigation/reportingTaskList/ReportingTaskList";
+
+const data: TaskListElement[] = [
+  {
+    type: "Page",
+    title: "main page element",
+  },
+  {
+    type: "Section",
+    title: "Facility 1 info",
+    elements: [
+      { type: "Page", title: "Review information", isChecked: true },
```

Comment:
Is the idea with `isChecked` to update this JSON object when a page is submitted & passes validation? ie: as part of the "submit' action on each page?

Reply:
That's up to the user! Something like that? We might want to hash out the detailed logic with @suhafa.  The AC for this card is "We need to be able to display a green checkmark" 😋